src/ddmd/backend/cod1.c

```diff
@@ -1998,8 +2001,9 @@ void getClibInfo(unsigned clib, symbol **ps, ClibInfo **pinfo)
                 }
                 else if (I32 && config.objfmt == OBJ_MSCOFF)
                 {
-                    s = symboly("_ms_aullrem", ALLREGS);
-                    cinfo->retregs32 = mDX|mAX;
+                    // use implementation in rt.llmath
+                    s = symboly("__ULDIV__", mAX|mBX|mCX|mDX);
```

Comment:
@WalterBright : the OpenBSD/Solaris version also calls signed version instead, seems like a typo. see https://github.com/dlang/dmd/blob/master/src/ddmd/backend/cod1.c#L1995

Reply:
I think you're right.